packages/ui/src/documents/CreateDocument.tsx

```diff
@@ -78,31 +86,30 @@ export const CreateDocument: FunctionComponent<Props> = (props) => {
     loadData();
   }, [loadData]);
 
-
   const handleOnSubmit = async (document: Document) => {
     setState({
       defaultDocument: document,
     });
-      try {
-        document = {
-          ...document,
-          attributes: {
-            ...document.attributes,
-            [HARDCODED_FIELDS.ORIGINATOR]: {
-              type: 'bytes',
-              value: user?.account,
-            } as any
-          },
-        };
-
-        let createResult: Document;
-        if (document.template && document.template !== '') {
-          createResult = (await httpClient.documents.clone(document)).data;
-        } else {
-          createResult = (await httpClient.documents.create(document)).data;
-        }
-        push(documentRoutes.index);
-        /*await httpClient.documents.update({
+    let createResult: Document | undefined;
+    try {
+      document = {
+        ...document,
+        attributes: {
+          ...document.attributes,
+          [HARDCODED_FIELDS.ORIGINATOR]: {
+            type: 'bytes',
+            value: user?.account,
+          } as any,
+        },
+      };
+
+      if (document.template && document.template !== '') {
+        createResult = (await httpClient.documents.clone(document)).data;
+      } else {
+        createResult = (await httpClient.documents.create(document)).data;
+      }
+      push(documentRoutes.index);
+      /*await httpClient.documents.update({
```

Comment:
I think this can be removed, right?

Reply:
Nope...this is just a temp comment so we can create a doc and test some stuff that I added. When the node is fixed this has to be uncommented. So the idea here is that we need to set the attribute for the NFT and we can set that only as an update because the values is the document_id. At the moment the update as some problems